src/Drupal/Commands/pm/PmCommands.php

```diff
@@ -87,6 +99,8 @@ public function enable(array $modules)
             }
         }
 
+        $this->validateInstallModules($todo);
```

Comment:
- You can remove this line and instead add `@hook validate pm:enable` at top of validateInstallModules method. 
- I think this method should be called `validateEnableModules`